android/app/src/main/java/com/reactnativenavigation/views/sharedElementTransition/SharedElementTransition.java

```diff
@@ -80,7 +80,7 @@ private void saveTextViewSpannedText(final TextView view) {
             @Override
             public void run() {
                 if (spannableText == null) {
-                    spannedText = (SpannedString) view.getText();
+                    spannedText = new SpannedString(view.getText());
```

Comment:
The exception isn't thrown when you wrap the Text in a transparent view since in that case We regard the shared element as a simple view - which doesn't have a text color property so this entire logic isn't executed.

I wonder under which condition `getText()` returns a builder and when does it return the concrete SpannedString. Can you confirm this has no performance hit?